### PR TITLE
Keep OS X CFNetwork from being identified as iOS

### PR DESCRIPTION
**Pull request by seandilda**
Some newer CFNetwork applications on OS X were being detected as running iOS and/or being an iOS-Device.  This prevents those false detections.

### CODE REVIEW

**Comment by commenthol** [comment]
@seandilda Thanks for the previous PR. Can you please rebase to master?

**Comment by seandilda** [comment]
Conflicts are resolved

**Comment by commenthol** [comment]
Thanks

File: regexes.yaml [hunk 1]
```diff
@@ -4430,6 +4446,9 @@ device_parsers:
     device_replacement: '$1$2,$3'
     brand_replacement: 'Apple'
     model_replacement: '$1$2,$3'
+  # @note: newer desktop applications don't show device info
+  # This is here so as to not have them recorded as iOS-Device
+  - regex: 'CFNetwork/.* Darwin/\d+\.\d+\.\d+ \(x86_64\)'
```
Comment (Plasma) [inline regexes.yaml]: This is causing blank Device Family information I think, see https://github.com/ua-parser/uap-csharp/issues/23

Comment (commenthol) [inline regexes.yaml]: Agreed #241 should fix this

**Pull request merged**